pkgs/by-name/tp/tproxy/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "tproxy";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "kevwan";
+    repo = "tproxy";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

Helps to avoid branch/tag confusion.